lib/matplotlib/figure.py

```diff
@@ -1265,21 +1265,21 @@ def subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
                 - 'col': each subplot column will share an x- or y-axis.
 
             When subplots have a shared x-axis along a column, only the x tick
-            labels of the bottom subplot are visible.  Similarly, when
-            subplots have a shared y-axis along a row, only the y tick labels
-            of the first column subplot are visible.
+            labels of the bottom subplot are created. Similarly, when subplots
+            have a shared y-axis along a row, only the y tick labels of the
+            first column subplot are created. To later turn other subplots'
+            ticklabels on, use :meth:`~matplotlib.axis.Axis.set_tick_params`.
 
-        squeeze : bool, default: True
-            - If True, extra dimensions are squeezed out from the returned
-              axis object:
+        squeeze : bool, optional, default: True
+            - If True, extra dimensions are squeezed out from the returned Axes
+              object:
```

Comment:
Actually, dimensions are squeezed out of the ndarray of Axes objects, not out of any such object.